plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/datatype/helper/InternalAttributeDeclarations.java

```diff
@@ -53,10 +53,12 @@ private static boolean addAttribbuteDeclarationToResource(final AttributeDeclara
 			LibraryElementTags.SUBAPP_REPRESENTATION_ATTRIBUTE, ElementaryTypes.BOOL);
 	public static final AttributeDeclaration RETAIN = createAttributeDeclaration(LibraryElementTags.RETAIN_ATTRIBUTE,
 			ElementaryTypes.USINT);
+	public static final AttributeDeclaration INOUTVISIBLEOUT = createAttributeDeclaration(
```

Comment:
please remove this here. as written above the attribute should never be in the model during normal operation.